workers/unity/Packages/com.improbable.gdk.core/Config/RuntimeConfig.cs

```diff
@@ -29,4 +31,40 @@ public static class RuntimeConfigNames
         public const string WorkerId = "workerId";
         public const string WorkerType = "workerType";
     }
+
+    /// <summary>
+    ///     Stores the configuration needed to connect via the Lcoator.
+    /// </summary>
```

Comment:
Given the docs focus.. could you add summary strings to the fields within these structs? 🙏